pkg/util/commons.go

```diff
@@ -184,6 +184,22 @@ func SanitizePhoneNumber(phoneNumber string) string {
 	return phoneNumber
 }
 
+// FormatOrderNumber formats the order number to xxxx-xxxx-xxxx
+func FormatOrderNumber(orderNumber string) string {
+	if len(orderNumber) < 12 {
+		return orderNumber
+	}
+
+	orderNumber = SanitizeOrderNumber(orderNumber)
+	return fmt.Sprintf("%s-%s-%s", orderNumber[:4], orderNumber[4:8], orderNumber[8:12])
```

Comment:
check for length here as well

Reply:
@abhyuday I think an additional length check may not be necessary as `SanitizeOrderNumber` function only removes "-" from the order number.

I want to sanitize the order number before formatting in case it has an incorrect "-" pattern.

Please let me know if we still need it.

Reply:
Yeah, I meant if someone passes 11----------11, there will be a crash

Reply:
@abhyuday Yes, that makes sense, thank you. I have made the update.

Reply:
cool, this you push the latest code.